tests/tests/tier0/monitor-system-status/test_monitor_system_status.py

```diff
@@ -1,35 +1,66 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
 import os
+import logging
 import time
-from typing import Dict
+from typing import Dict, List
 
 from bluechi_test.util import read_file
 from bluechi_test.test import BluechiTest
 from bluechi_test.container import BluechiControllerContainer, BluechiNodeContainer
 from bluechi_test.config import BluechiControllerConfig, BluechiNodeConfig
 
 
+LOGGER = logging.getLogger(__name__)
+
 node_one = "node-1"
 node_two = "node-2"
 node_three = "node-3"
 nodes = [node_one, node_two, node_three]
 
 
 def stop_all_agents(nodes: Dict[str, BluechiControllerContainer]):
+    LOGGER.info("Stopping all agents...")
```

Comment:
debug?

Reply:
I think these logs are useful to have in case of a test failure - and we should have more. I'd rather use info logs by default for integration tests (at least now) and change to debug if those become too much and confusing.

Reply:
I changed a few logs to debug, but want to keep the log for the events at info level.